mbof/views.py

```diff
@@ -66,9 +68,37 @@ class EventViewSet(OcellusViewSet):
     queryset = Event.objects.all().order_by('postingTime')
     serializer_class = EventSerializer
 
+    def retrieve(self, request, *args, **kwargs):
+        log_action(request)
+        instance = self.get_object()
+        serializer = self.get_serializer(instance)
+        serializer_data = serializer.data
+        if serializer_data['owner'] == self.request.user.username:
+            serializer_data['owner'] = "True"
+        else:
+            serializer_data['owner'] = "False"
+        return Response(serializer_data)
```

Comment:
My python is weak.  Is this returning the data in either case just marking it with whether or not it was requested by the owner?